wiki/redirect/osu!academy/de.md

```diff
@@ -0,0 +1 @@
+/wiki/Announcements/osu!academy/?locale=de
```

Comment:
alrighty, I will remove it but I want to first see if _that_ (see my comment above) works or not.
This is also why a couple of the redirects are in other languages as well I suppose.